packages/experimental/Button/SPEC.md

```diff
@@ -99,13 +99,19 @@ export type ButtonProps = Omit<IWithPressableOptions<ViewProps>, 'onPress'> & {
    */
   iconPosition?: 'before' | 'after';
 
-  // /**
-  //  * A button can show a loading indicator if it is waiting for another action to happen before allowing itself to
-  //  * be interacted with.
-  //  * @default false
-  //  */
+  /**
+   * A button can show a loading indicator if it is waiting for another action to happen before allowing itself to
+   * be interacted with.
+   * @default false
+   */
   loading?: boolean;
 
+  /**
+   * A RefObject to access the IButton interface. Use this to access the public methods and properties of the component.
+   * Note that this does not allow for a ref callback to be passed in.
+   */
+  ref?: React.RefObject<IFocusable>;
```

Comment:
Will this be a breaking change?

Reply:
Whoops, need to edit that sentence, since it's not true anymore.

It wouldn't be a breaking change, though - componentRef works the same way.